tests/logic.js

```diff
@@ -113,6 +113,14 @@ require([
                 });
             });
         });
+
+        describe("should not add validation error for", function() {
+            it("meta reference", function () {
```

Comment:
How is this testing meta references? A form with a single text question doesn't have any paths with meta in them (I thought...).